<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelDinheiro">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dinheiro (Em milh√µes):</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPessoa">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Pessoas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRisco">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Risco:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResp">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Risco Resp</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinDinheiro">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinPessoa">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalcular">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
